src/index.js

```diff
@@ -85,14 +86,16 @@ render(
     <ReduxProvider store={store}>
       <BrowserRouter>
         <CookiesProvider>
-          <Switch>
-            <Route path="/accounts" component={Accounts} />
-            <ProtectedRoute exact path="/" component={ProgramList} />
-            <ProtectedRoute exact path="/programs" component={ProgramList} />
-            <ProtectedRoute exact path="/mission-control" component={MissionControl} />
-            <ProtectedRoute exact path="/user/settings" component={UserSetting} />
-            <Route component={NotFound} />
-          </Switch>
+          <ThemeProvider theme={responsiveFontSizes(theme)}>
```

Comment:
`responsiveFontSizes` will auto adjust our relative font sizes based on screen width. They can also be set manually for different screen widths, so we'll have to see how we like the defaults.

![image](https://user-images.githubusercontent.com/976857/80670467-f507f400-8a74-11ea-9d59-864cab74b937.png)